woocommerce/woocommerce-ios
# [Mobile Payments] [Stripe Backend] Bump minimum version of stripe plugin to 6.1.0

Closes: #5733 
Closes: #5921 

@ctarda @joshheald - just one review needed but either of y'all are welcome

### Description
- The WooCommerce Stripe Gateway plugin version 6.1.0 will be released next week
- When that becomes available, this can be tested to ensure we 1) require it and 2) read the live and test mode booleans that it adds correctly

cc @malinajirka for Android to follow suit

### Testing instructions
- Install 6.1.0 on your store
- Briefly edit the plugin header to say 6.0.0
- Ensure you are prompted to update the plugin
- Edit the plugin header back to 6.1.0
- Ensure the live and test mode booleans are correctly decoded by using breakpoints

OR

- By default, most of our test sites will have is_live false and test_mode true and, assuming the (test) account has been setup, you should be able to connect to a reader (note: sites with is_live false will always have test_mode true - they can't have test_mode false)
- So, if you don't have access to a live site, you can hack `includes/admin/class-wc-rest-stripe-account-controller.php` on your test site to return is_live true. This will allow you to test live sites in test mode, which should be detected and you should see a warning that it isn't supported. If you don't have access to a live site, you won't be able to test is_live true and test_mode false, and that's ok - testing the booleans independently should suffice.

### Screenshots
- N/A

---
